src/dev/impl/DevToys/DevToys.csproj

```diff
@@ -562,6 +574,9 @@
     <PackageReference Include="Microsoft.Toolkit.Uwp.UI.Controls" Version="$(Nuget_MicrosoftToolkitUwpUIControls)" />
     <PackageReference Include="Microsoft.UI.Xaml" Version="$(NuGet_MicrosoftUIXaml)" />
     <PackageReference Include="Newtonsoft.Json" Version="$(NuGet_NewtonsoftJson)" />
+    <PackageReference Include="NLipsum" Version="$(NuGet_NLipsum)">
+      <Version>1.1.0</Version>
```

Comment:
You should be able to remove this line, since you set the version number on the line above :)

Reply:
Ah good catch! Whoops! Fixed it now :)